pallets/xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -0,0 +1,242 @@
+
+		<AssetTransactorOf<T>>::deposit_asset(&asset, &sender_location).unwrap();
+		// check the assets of origin.
+		assert!(!T::TransactAsset::balance(&sender_account).is_zero());
+
+		let mut executor = new_executor::<T>(sender_location);
+		executor.holding = worst_case_holding.into();
+		let instruction = Instruction::<XcmCallOf<T>>::WithdrawAsset(vec![asset.clone()].into());
+		let xcm = Xcm(vec![instruction]);
+	}: {
+		executor.execute(xcm)?;
+	} verify {
+		// check one of the assets of origin.
+		assert!(T::TransactAsset::balance(&sender_account).is_zero());
+		assert!(executor.holding.ensure_contains(&vec![asset].into()).is_ok());
+	}
+
+	transfer_asset {
+		let (sender_account, sender_location) = account_and_location::<T>(1);
+		let asset = T::get_multi_asset();
+		let assets: MultiAssets = vec![ asset.clone() ].into();
+		// this xcm doesn't use holding
+
+		let dest_location = T::valid_destination()?;
+		let dest_account = T::AccountIdConverter::convert(dest_location.clone()).unwrap();
+
+		<AssetTransactorOf<T>>::deposit_asset(&asset, &sender_location).unwrap();
+		assert!(T::TransactAsset::balance(&dest_account).is_zero());
+
+		let mut executor = new_executor::<T>(sender_location);
+		let instruction = Instruction::TransferAsset { assets, beneficiary: dest_location };
+		let xcm = Xcm(vec![instruction]);
+	}: {
+		executor.execute(xcm)?;
+	} verify {
+		assert!(T::TransactAsset::balance(&sender_account).is_zero());
+		assert!(!T::TransactAsset::balance(&dest_account).is_zero());
+	}
+
+	transfer_reserve_asset {
+		let (sender_account, sender_location) = account_and_location::<T>(1);
+		let dest_location = T::valid_destination()?;
+		let dest_account = T::AccountIdConverter::convert(dest_location.clone()).unwrap();
+
+		let asset = T::get_multi_asset();
+		<AssetTransactorOf<T>>::deposit_asset(&asset, &sender_location).unwrap();
+		let assets: MultiAssets = vec![ asset ].into();
+		assert!(T::TransactAsset::balance(&dest_account).is_zero());
+
+		let mut executor = new_executor::<T>(sender_location);
+		let instruction = Instruction::TransferReserveAsset {
+			assets,
+			dest: dest_location,
+			xcm: Xcm::new()
+		};
+		let xcm = Xcm(vec![instruction]);
+	}: {
+		executor.execute(xcm)?;
+	} verify {
+		assert!(T::TransactAsset::balance(&sender_account).is_zero());
+		assert!(!T::TransactAsset::balance(&dest_account).is_zero());
+		// TODO: Check sender queue is not empty. #4426
+	}
+
+	reserve_asset_deposited {
+		let (trusted_reserve, transferable_reserve_asset) = T::TrustedReserve::get()
+			.ok_or(BenchmarkError::Skip)?;
+
+		let assets: MultiAssets = vec![ transferable_reserve_asset ].into();
+
+		let mut executor = new_executor::<T>(trusted_reserve);
+		let instruction = Instruction::ReserveAssetDeposited(assets.clone());
+		let xcm = Xcm(vec![instruction]);
+	}: {
+		executor.execute(xcm).map_err(|_| {
+			BenchmarkError::Override(
+				BenchmarkResult::from_weight(T::BlockWeights::get().max_block)
```

Comment:
Why does this have an override?

Reply:
This is mimicking whatever is happening in polkadot repo. In reality it does not matter, because none of the assets parachains have trusted reserve, so this benchmark is skipped altogether.

Reply:
@KiChjang I believe this code has actually been introduced by you in the polkadot repo, I'd be very happy to hear what the motivation was.

Reply:
I'm going to remove this override as the whole bench is skipped anyway.

Reply:
done